cfme/tests/infrastructure/test_infra_object_tag_visibility.py

```diff
@@ -0,0 +1,66 @@
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.cluster import Cluster
+from cfme.infrastructure.host import Host
+from cfme.infrastructure.datastore import Datastore
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.infrastructure.virtual_machines import Vm, Template
+from fixtures.provider import setup_one_or_skip
+from utils.providers import ProviderFilter
+
+
+pytestmark = [test_requirements.tag, pytest.mark.tier(2)]
+
+
+@pytest.fixture(scope='module')
+def a_provider(request):
+    prov_filter = ProviderFilter(classes=[VMwareProvider])
```

Comment:
Maybe, in this case, such restriction is unnecessary. Any infra provider will be good for me